images/metering-ansible-operator/roles/meteringconfig/tasks/deploy_resources.yml

```diff
@@ -57,3 +57,4 @@
     loop_var: resource
     label: "{{ resource.template_file }}"
   when: not (resource.create | default(true))
+  no_log: true
```

Comment:
still want to be able to log when resources are marked as candidates to be pruned